src/main/java/net/coderbot/iris/gl/shader/StandardMacros.java

```diff
@@ -150,6 +151,13 @@ public static String group(Matcher matcher, String name) {
 		return Arrays.stream(extensions).map(s -> "MC_" + s).collect(Collectors.toList());
 	}
 
+	public static List<String> getIrisDefines() {
+		List<String> defines = new ArrayList<>();
+		defines.add("dragonDeathProgressSupported");
```

Comment:
We should choose a name that is more in line with the other standard macros.
```suggestion
		defines.add("MC_UNIFORM_DRAGON_DEATH_PROGRESS");
```